PlottingAnalysis/TS_TopFeatures.m

```diff
@@ -83,6 +85,15 @@
 default_numNulls = 0; % by default, don't compute an empirical null distribution
                       % by randomizing class labels
 addParameter(inputP,'numNulls',default_numNulls,@isnumeric);
+% numFolds:
```

Comment:
Why do you need to run cross-validation for *single* features? The risk of overfitting in one dimension is negligible…

Reply:
I figured it might be useful in the event you have a very small number of time series. We can remove it if you'd like?